### PR TITLE
AST-1909 - Compatibility with WordPress 6.0 - Image block not aligned center for old users when Legacy option set from the Block Editor.

### PR DESCRIPTION
**Pull request by nileshc-bsf**
### Description
The image block is not centred aligned for WP6.0

Astra customizer settings - https://share.bsf.io/7Ku6yByx 

Steps:

- Import the above settings on your end
- Create  a page/post and add an image block and make it as center aligned
- Publish and check on frontend - It will not center align

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -1312,6 +1305,19 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 					$parse_css .= astra_parse_css( $mobile_screen_max_gb_css, '', astra_get_mobile_breakpoint() );
 				}
 
+				$is_legacy_setup = ( 'legacy' === astra_get_option( 'wp-blocks-ui', 'custom' ) || true === astra_get_option( 'blocks-legacy-setup', false ) ) ? true : false;
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:1308]:
```suggestion
				$is_legacy_setup = ( 'legacy' === astra_get_option( 'wp-blocks-ui' ) ) ? true : false;
```

Comment (nileshc-bsf) [reply inc/class-astra-dynamic-css.php]: Done

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
+v3.8.3 (Unreleased)
+- Fix: Compatibility with WordPress 6.0 - Image block not aligned center for old users when Legacy option set from the Block Editor.
```
Comment (imnavanath) [inline changelog.txt:2]:
```suggestion
- Fix: Image block not aligned centrally for Legacy block editor setup.
```

**Review by imnavanath — APPROVED** [review]

File: changelog.txt (hunk 2 above)
Comment (nileshc-bsf) [reply changelog.txt]: Done

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**